nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/FileTransfer.java

```diff
@@ -39,6 +39,8 @@
 
     void flush() throws IOException;
 
+    boolean flush(FlowFile flowFile) throws IOException;
```

Comment:
Similar to `flush()`, other methods those use `getClient` (in FTPTransfer) or `getChannel` (in SFTPTransfer) should pass incoming FlowFile in order to evaluate EL properly. [NIFI-3590](https://issues.apache.org/jira/browse/NIFI-3590) is essentially the same to NIFI-3281 which is caused by not passing FlowFile.

Do we want to address it, too? Ideally, the codes `getClient(null)` and `getChannel(null)` should be removed completely.